alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -428,6 +426,9 @@ private function initMembers()
 		
 		$this->bundleConfig = json_decode($confVars, true);
 		$this->mergeVersionsParamIntoConfig();
+		if (!$this->bundleConfig) {
```

Comment:
mergeVersionsParamIntoConfig uses the bundle config ($this->bundleConfig = array_merge($this->bundleConfig, $versionsArr);) so you may want to move this check up. 

If bundleConfig in null and you will call array_merge it will result in a warning.

Another option is to define the bundleConfig to default value of an empty array and change this check accordingly.